### PR TITLE
upgrade version of boto3 and baram

### PR DESCRIPTION
**Pull request by alprinf**
upgrade boto3 and baram version
- boto3: 1.18.65 -> 1.26.74
- baram: 0.2.6 -> 0.2.7

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ readme = "readme.md"
 python = "^3.7.10"
 fire = "^0.4.0"
 pytest-cov = "^3.0.0"
-boto3 = "1.18.65"
+boto3 = "1.26.74"
 aiohttp = "^3.8.1"
 nest-asyncio = "^1.5.5"
 toml = "^0.10.2"
```
Comment (Baescott) [inline pyproject.toml:15]: You have to update `baram`'s version also. It is located above these.

**Review by Baescott — CHANGES_REQUESTED** [review]

**Review by Baescott — APPROVED** [review]
lgtm

File: pyproject.toml (hunk 1 above)
Comment (alprinf) [reply pyproject.toml:15]: thanks for pointing out. I updated baram to 0.2.7 in last commit.

**Review by lks21c — APPROVED** [review]

**Comment by lks21c** [comment]
@alprinf go ahead!

**Pull request merged**